crates/sui-graphql-rpc/src/types/base64.rs

```diff
@@ -10,6 +10,7 @@ use fastcrypto::encoding::Encoding as FastCryptoEncoding;
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub(crate) struct Base64(pub(crate) Vec<u8>);
 
+/// String containing a Base64-encoded binary data.
```

Comment:
nit, grammar (data is plural).

```suggestion
/// String containing Base64-encoded binary data.
```